egs2/TEMPLATE/asr1/pyscripts/utils/extract_spk_embed_parallel.py

```diff
@@ -0,0 +1,401 @@
+#!/usr/bin/env python3
```

Comment:
Could you please also update the tts.sh correspondingly to use the parallel version (or let user to select which version to use)?